GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -205,6 +205,106 @@ extension Glia {
             debugPrint("💥 Core SDK configuration is not valid. Unexpected error='\(error)'.")
         }
     }
+
+    /// Deprectated, use ``Glia.startEngagement(engagementKind:in queueIds:theme:features:sceneProvider:)`` instead.
+    /// Use ``configure(with configuration:uiConfig:assetsBuilder:completion:)`` to pass in ``RemoteConfiguration``.
+    @available(*, deprecated, message: """
+            Deprectated, use ``Glia.startEngagement(engagementKind:in queueIds:theme:features:sceneProvider:)`` instead.
+            Use ``configure(with configuration:uiConfig:assetsBuilder:completion:)`` to pass in ``RemoteConfiguration``.
+        """
+    )
+    public func startEngagementWithConfig(
+        engagement: EngagementKind,
+        in queueIds: [String],
+        uiConfig: RemoteConfiguration,
+        assetsBuilder: RemoteConfiguration.AssetsBuilder = .standard,
+        features: Features = .all,
+        sceneProvider: SceneProvider? = nil
+    ) throws {
+        // Store assetsBuilder and apply remote config to have ability to
+        // apply them for Call Visualizer flow if integrator use
+        // old `configure` method
+        theme.apply(configuration: uiConfig, assetsBuilder: assetsBuilder)
+        self.assetsBuilder = assetsBuilder
+
+        if let config = configuration {
+            theme.showsPoweredBy = !config.isWhiteLabelApp
+            theme.chat.connect.queue.firstText = config.companyName
+            theme.call.connect.queue.firstText = config.companyName
+        }
+
+        try startEngagement(
+            engagementKind: engagement,
+            in: queueIds,
+            theme: theme,
+            features: features,
+            sceneProvider: sceneProvider
+        )
+    }
+
+    /// Deprectated, use ``Glia.startEngagement(engagementKind:in queueIds:features:sceneProvider:)`` instead.
+    /// Use ``configure(with configuration:uiConfig:theme:assetsBuilder:completion:)`` to pass in ``RemoteConfiguration``.
+    @available(*, deprecated, message: """
+            Deprectated, use ``Glia.startEngagement(engagementKind:in queueIds:features:sceneProvider:)`` instead.
+            Use ``configure(with configuration:uiConfig:theme:assetsBuilder:completion:)`` to pass in ``RemoteConfiguration``.
+        """
+    )
+    public func startEngagement(
+        engagementKind: EngagementKind,
+        in queueIds: [String],
+        theme: Theme = Theme(),
+        features: Features = .all,
+        sceneProvider: SceneProvider? = nil
+    ) throws {
+        let trimmedQueueIds = queueIds
```

Comment:
Could it simplify by setting `self.theme = theme` and calling new `startEngagement` method?

Reply:
fixed [6aa5da3](https://github.com/salemove/ios-sdk-widgets/pull/835/commits/6aa5da3f7dc7630e7d7754440177e2a97b077103)